src/kagglehub/env.py

```diff
@@ -33,3 +42,30 @@ def read_kaggle_build_date() -> str:
     except FileNotFoundError:
         logging.warning(f"Build date file {build_date_file} not found in Kaggle Notebook environment.")
         return "unknown"
+
+
+def search_lib_in_call_stack(lib_name: str) -> Optional[str]:
+    """Search the call stack for a given library name and get its information.
+
+    Args:
+        lib_name (str):
+            The name of the library to search for.
+            We use re.match so the lib_name must match the module name from beginning.
```

Comment:
Could you use `module_name.startswith(libname)`?

https://www.w3schools.com/python/ref_string_startswith.asp

Reply:
Done. Thank you for the quick review!